### PR TITLE
Adding the add_from_raster function

### PR DESCRIPTION
**Pull request by choisy**

### CODE REVIEW

File: R/add_from_raster.R [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#'
+#' @importFrom magrittr %<>%
+#' @importFrom sp spTransform proj4string
+#' @importFrom raster extract
+#'
+#' @export
+#'
+#' @examples
+#' library(sp)
+#' sptools::gadm("vietnam", "sp", 0) %>%
```
Comment (LucieContamin) [inline R/add_from_raster.R]: Can I suggest to write the example like this: 
``` r
#' library(sp)
#' library(magrittr) # for " %>% "
#'
#' value <- sptools::gadm("vietnam", "sp", 0) %>%
#'   spsample(100, "random") %>%
#'   SpatialPointsDataFrame(data.frame(variable = 1:100)) %>%
#'   add_from_raster(srtmVN::getsrtm(), "elevation")
#'
#' value %>%
#'   slot("data") %>%
#'   head()
```
To make it more clear?

**Review by LucieContamin — APPROVED** [review]
Cool function

File: R/add_from_raster.R (hunk 1 above)
Comment (choisy) [reply R/add_from_raster.R]: Sure! Good idea!

Comment (choisy) [reply R/add_from_raster.R]: Would you please do it?

**Comment by LucieContamin** [comment]
The example has been updated.

**Pull request merged**